onnxmltools/convert/sklearn/shape_calculators/KNeighborsClassifier.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------
```

Comment:
It looks like this file is not necessary as well. Could you reuse [this shape calculator](https://github.com/onnx/onnxmltools/blob/0cd78a742837aa429c57d3a54f269a2421770cca/onnxmltools/convert/sklearn/shape_calculators/LinearClassifier.py#L69) by moving `



 
register_shape_calculator('SklearnKNeighborsClassifier', calculate_sklearn_k_neighbors_classifier_output_shapes)`?

Reply:
Done